src/status_im/protocol/web3/inbox.cljs

```diff
@@ -44,14 +44,40 @@
                                                   (swap! peers conj enode)
                                                   (success-fn result))))))
 
-(defn mark-trusted-peer [web3 enode success-fn error-fn]
+(defn extract-enode-id [enode]
+  (-> enode
+      env.utils/get-host
+      (string/split "@")
+      (get 0)))
+
+(defn registered-peer? [peers enode]
+  (->> enode
+       extract-enode-id
+       (contains?
```

Comment:
Would put on some line / possible extract into separate fn name (can be let in this one)

Reply:
Agree. Refactored that fn a bit.